### ISSUE TITLE
Add to CPAN

### ISSUE THREAD

**Issue by okurz** [okurz/openqa_irc#8]

**Comment by Soulofdestiny** [comment]
here's a good point to start with that:
http://bit.ly/1PyELyO

**Comment by okurz** [comment]
yes, looks good. thank you. First I assume we need some build system, e.g. Build.PL or Makefile.PL or similar. There are some scripts which create a project directory structure which we could use.
